HelpSource/Classes/UnitTest.schelp

```diff
@@ -82,98 +82,133 @@ TestUnitTest.new.run;
 ::
 
 
-
-
-
 METHOD:: runTestMethod
 Run a single test method of this class
 
-ARGUMENT:: method
-the method to run
-
 code::
 TestUnitTest.new.runTestMethod(TestUnitTest.findMethod(\test_assert));
 ::
 
+ARGUMENT:: method
+the method to run
+
 returns:: (describe returnvalue here)
 
 PRIVATE:: assert, assertArrayFloatEquals, asynchAssert, tearDown, setUp, assertEquals, failed, passed, assertFloatEquals, s, ifAsserts, findTestMethods, wait, bootServer
 
 
-SECTION:: Writing tests by subclassing UnitTest
+METHOD:: assert
+
+Asserts that an expression must be true.
 
 code::
-YourClass.test
+this.assert(2 == 2, "passes");
+this.assert(2 == 2.00001, "fails");
 ::
-Runs the test class for code::YourClass::, which is assumed to be named code::TestYourClass::.
 
-If no test class if found it politely declines.
+argument:: test
+A boolean, where code::true:: means that the test is passed.
 
-note::
-UnitTests for the Common library classes are kept in the CommonTests quark. 	This enables you to easily install and uninstall these tests.
-::
+argument:: message
+A message describing the purpose of the assertion, e.g. code::"foo
+should be less than bar"::.
+Posted if code::report:: is true.
 
+argument:: report
+Reports the result of the test if code::true:: (default is code::true::)
```

Comment:
You don't need to remove it, but stating the default value here is redundant since this will already be shown above.